src/Init/TacticsExtra.lean

```diff
@@ -41,4 +41,24 @@ macro_rules
   | `(tactic| if%$tk $c then%$ttk $pos else%$etk $neg) =>
     expandIfThenElse tk ttk etk pos neg fun pos neg => `(if h : $c then $pos else $neg)
 
+/--
+Rewrite with the given rules and normalize casts between steps.
```

Comment:
```suggestion
Rewrites with the given rules, normalizing casts prior to each step.
```
Indicative mood, and as I read the code, it's not just between, but also at the beginning.